script/nnbd_plugins.sh

```diff
@@ -22,6 +22,7 @@ readonly NNBD_PLUGINS_LIST=(
   "url_launcher"
   "video_player"
   "webview_flutter"
+  "shared_preferences_windows"
```

Comment:
I wouldn't expect this to be necessary; everything else here is app-facing. Did you do this to resolve a CI failure?

Reply:
Yes, had a previous PR failing for this reason. So added this.

Reply:
I don't think it is necessary or needed to add it here. You just have a file not complying with the expected format.

Reply:
@gaetschwartz, which one is not complying?

Reply:
Read the checks.  It says :

> These files are not formatted correctly (see diff below):
  shared_preferences/shared_preferences_windows/lib/shared_preferences_windows.dart